spec/features/admin/bulk_order_management_spec.rb

```diff
@@ -469,17 +469,17 @@
         expect(page).to have_no_selector "tr#li_#{li4.id}"
       end
 
-      it "displays only line items whose orders meet the date restriction criteria, when changed" do
+      it "displays only line items whose orders meet the date restriction criteria, when changed", :debug do
```

Comment:
what's that `:debug` argument?

Reply:
It's a tag that allow the tester to test only tests tagged with this. 
https://relishapp.com/rspec/rspec-core/v/2-4/docs/command-line/tag-option
And I forgot to delete this one... Thanks!